docs-2.0/5.configurations-and-logs/2.log-management/logs.md

```diff
@@ -0,0 +1,63 @@
+# Logs
+
+**Nebula Graph** uses [glog](https://github.com/google/glog) to print logs, uses [gflag](https://gflags.github.io/gflags/) to control the severity level of the log, and provides an HTTP interface to dynamically change the log level at runtime to facilitate tracking.
+
+## Log Directory
+
+The default log directory is `/usr/local/nebula/logs/`.
+
+> **NOTE**: If you deleted the log directory during runtime, the runtime log would not continue to be printed. However, this operation will not affect the services. Restart the services to recover the logs.
+
+## Parameter Description
+
+### Two most commonly used flags in glog
+
+- minloglevel: The scale of minloglevel is 0-4. The numbers of severity levels INFO(DEBUG), WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively. Usually specified as 0 for debug, 1 for production. If you set the minloglevel to 4, no logs are printed.
+- v: The scale of v is 0-3. When the value is set to 0,  you can further set the severity level of the debug log. The greater the value is, the more detailed the log is.
+
+### Configuration Files
+
+The default severity level for the metad, graphd, and storaged logs can be found in the configuration files (usually in `/usr/local/nebula/etc/`).
+
+## Check and Change the Severity Levels Dynamically
+
+Check all the flag values (log values included) of the current gflags with the following command. `curl` is only available when the `local_config` parameter is set to `true`.
```

Comment:
`curl` will always change the log level at runtime.

Add a note: If you change the runtime log level, when you restart the service, the log level depends on the configure file specifications. See configure file specifications doc.